index.html

```diff
@@ -0,0 +1,346 @@
+        <img
+          src="./images/Snapshoot-Portfolio2.svg"
+          alt=""
+          class="showcase-img"
+        />
+        <h2>Multi-Post Stories</h2>
+        <div class="article">
+          <p class="canopy">CANOPY</p>
+          <img
+            src="./images/Counter.svg"
+            alt=""
+          />
+          <p>Back End DEv</p>
+          <img
+            src="./images/Counter.svg"
+            alt=""
+          />
+          <p>2015</p>
+        </div>
+        <p>
+          A daily selection of privately
+          personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="available-articles">
+          <a href="#">html</a>
+          <a href="#">css</a>
+          <a href="#">javaScript</a>
+        </div>
+        <button type="button" class="action-btn">
+          See Project
+        </button>
+      </section>
+
+      <!-- end of showcase2 -->
+
+      <section class="section2 d-flex2">
+        <img
+          src="./images/Snapshoot-Portfolio3.svg"
+          alt=""
+          class="showcase-img"
+        />
+        <h2>Tonic</h2>
+        <div class="article">
+          <p class="canopy">CANOPY</p>
+          <img
+            src="./images/Counter.svg"
+            alt=""
+          />
+          <p>Back End DEv</p>
+          <img
+            src="./images/Counter.svg"
+            alt=""
+          />
+          <p>2015</p>
+        </div>
+        <p>
+          A daily selection of privately
+          personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="available-articles">
+          <a href="#">html</a>
+          <a href="#">css</a>
+          <a href="#">javaScript</a>
+        </div>
+        <button type="button" class="action-btn">
+          See Project
+        </button>
+      </section>
+
+      <!-- end of showcase3 -->
+
+      <section class="section2 d-flex2">
+        <img
+          src="./images/art-printing.svg"
+          alt=""
+          class="showcase-img"
+        />
+        <h2>Multi-Post Stories</h2>
+        <div class="article">
+          <p class="canopy">CANOPY</p>
+          <img
+            src="./images/Counter.svg"
+            alt=""
+          />
+          <p>Back End DEv</p>
+          <img
+            src="./images/Counter.svg"
+            alt=""
+          />
+          <p>2015</p>
+        </div>
+        <p>
+          A daily selection of privately
+          personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="available-articles">
+          <a href="#">html</a>
+          <a href="#">css</a>
+          <a href="#">javaScript</a>
+        </div>
+        <button type="button" class="action-btn">
+          See Project
+        </button>
+      </section>
+      <!-- end of showcase4 -->
+    </article>
+    <!-- end of section -->
+
+    <!-- beginning of about -->
+    <article>
```

Comment:
- [ ] As mentioned above, please use the `section` tag in place of this `article` tag. They are more semantic and is better for SEO purposes and accessibility